supreme/src/androidMain/kotlin/at/asitplus/signum/supreme/hazmat/InternalsAccessors.kt

```diff
@@ -0,0 +1,21 @@
+package at.asitplus.signum.supreme.hazmat
+
+import at.asitplus.signum.supreme.HazardousMaterials
+import at.asitplus.signum.supreme.os.AndroidKeystoreSigner
+import at.asitplus.signum.supreme.sign.AndroidEphemeralSigner
+import at.asitplus.signum.supreme.sign.EphemeralKey
+import at.asitplus.signum.supreme.sign.EphemeralKeyBase
+import at.asitplus.signum.supreme.sign.Signer
+import java.security.PrivateKey
+
+/** The underlying JCA [PrivateKey] object. */
+@HazardousMaterials
+val EphemeralKey.jcaPrivateKey get() = (this as? EphemeralKeyBase<*>)?.privateKey as? PrivateKey
+
+/** The underlying JCA [PrivateKey] object. Not available for unlocked KeyStore signers; see [jcaSignatureInstance]. */
+@HazardousMaterials
```

Comment:
is this still true and sensible after the rewrite?

Reply:
@iaik-jheher sorry, i was off a line. is it still potentially unavailable?

Reply:
it is not, comment was amended